lib/util.js

```diff
@@ -83,6 +83,26 @@ const uploadCRXFile = (endpoint, region, crxFile, componentId) => {
 
   mkdirp.sync(unzipDir)
 
+  return unzip(crxFile, unzipDir).then(() => {
+    const data = parseManifest(path.join(unzipDir, 'manifest.json'))
+    const id = componentId || getIDFromBase64PublicKey(data.key)
+    const version = data.version
+    const hash = generateSHA256HashOfFile(crxFile)
+    const name = data.name
+
+    return uploadExtension(id, version, crxFile)
+      .then(() => console.log(`Uploaded ${crxFile}`))
+      .catch((err) => {
+        throw err
+      })
+  })
+}
+
+const updateDBForCRXFile = (endpoint, region, crxFile, componentId) => {
+  const unzipDir = path.join('build', 'unzip', path.parse(crxFile).name)
+
+  mkdirp.sync(unzipDir)
+
   unzip(crxFile, unzipDir).then(() => {
```

Comment:
Perhaps return this `Promise` so that the caller can add an error handler to log, or at least wait until finished.